### DISCUSSION TITLE
Using shared <Video /> instance for fullscreen

### DISCUSSION THREAD

**Discussion by proohit** [TheWidlarzGroup/react-native-video#4317] [Q&A]
### What are you trying to do?

We're trying to implement a custom fullscreen video, because we need custom controls. We want some kind of "shared" `<Video />` instance so that switching to fullscreen doesn't reload the given media.

### What have you tried?

Rendering the fullscreen views. The problem is that anytime we render the fullscreen views, the player gets rerendered, causing the `onLoad` event to trigger, thus the video component reloads the media file.

**Comment by freeboub** [comment]
no, it is not possible to share instance.
The solution I used is to but the `<Video/> `outside of the screen stack when we can move it full screen over the screens.
